src/app/azureResource/components/azureResourceView.spec.tsx

```diff
@@ -0,0 +1,105 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import { AzureResourceView } from './azureResourceView';
+import { AccessVerificationState } from '../models/accessVerificationState';
+
+describe('AzureResourceView', () => {
+    it('matches snapshot when current resource is undefined', () => {
+        expect(shallow(
+            <AzureResourceView
+                activeAzureResource={undefined}
+                currentHostName="hostName"
+                currentUrl="url"
+                setActiveAzureResourceByHostName={jest.fn()}
+            />
+        )).toMatchSnapshot();
```

Comment:
is it possible to use the testSnapshot( ) helper here instead? We are trying to make the test pattern more consistent but haven't fully migrate all the tests which are still using pattern, but we are trying to make the new tests using the test helpers if possible.